.browserslistrc

```diff
@@ -0,0 +1,5 @@
+# Browsers that we support
+
+> 0.1%
+ie 8
+ie 9
```

Comment:
Not blocking this PR, but at some point we should update this list. A good reference would be to copy or use [Gutenberg's](https://github.com/WordPress/gutenberg/tree/master/packages/browserslist-config).

Reply:
Good point, perhaps we can pull in `@wordpress/browserslist-config` directly and customise if necessary.